apps/full-site-editing/full-site-editing-plugin/wpcom-block-editor-nav-sidebar/src/types.ts

```diff
@@ -0,0 +1,4 @@
+declare module '@wordpress/icons' {
+	export const chevronLeft: any;
+	export const wordpress: any;
+}
```

Comment:
Hmm, I thought `@wordpress/icons` came [with typedefs included](https://github.com/WordPress/gutenberg/blob/16d1e41c3f3fe72446d00a2f25fbe34257ae1c48/packages/icons/CHANGELOG.md#new-feature) these days :thinking:

Reply:
Yeah I need to get to the bottom of this. When I run `yarn list --pattern @wordpress/icons` it says I have a single copy: version 2.0.0. Which has the type info. But when I look at the `node_modules/@wordpress/icons/package.json` I have version 1.2.0 🙃 

That makes it seem like an issue with just my machine, but I feel like I've cleanly installed dependencies again ...

Reply:
`rm -rf node_modules` did the trick 😆